easybuild/easyconfigs/s/subread/subread-1.6.5-GCC-8.3.0.eb

```diff
@@ -0,0 +1,36 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'subread'
+version = '1.6.5'
+
+# HTTPS is not valid
+homepage = 'http://bioinf.wehi.edu.au/featureCounts/'
+description = """featureCounts is a highly efficient general-purpose read summarization program that counts mapped
+reads for genomic features such as genes, exons, promoter, gene bodies, genomic bins and chromosomal locations."""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = [('http://sourceforge.net/projects/%(name)s/files/%(name)s-%(version)s/', 'download')]
+sources = ['%(name)s-%(version)s-source.tar.gz']
+checksums = ['f4defb3debd01a67fb3d5219cddf9a518ed2e69405e26f3b8990d305bababd0d']
+
+start_dir = 'src'
+
+buildopts = " -f Makefile.Linux"
+
+files_to_copy = ['bin']
+
+local_binaries_list = [
+    'exactSNP', 'featureCounts', 'subindel', 'subjunc', 'sublong', 'subread-align', 'subread-buildindex',
+]
+sanity_check_paths = {
+    'files': ['bin/%s' % f for f in local_binaries_list],
+    'dirs': ['bin/utilities'],
+}
+
+runtest = 'fail; cd %(builddir)s/%(name)s-%(version)s-source/test && bash test_all.sh'
```

Comment:
The `fail; ` bit is a nasty hack, why not run the tests via `sanity_check_commands` instead?